azure-pipelines.yml

```diff
@@ -134,19 +127,12 @@ jobs:
     - task: CopyFiles@2
       displayName: Stage .NET artifacts
       inputs:
-        sourceFolder: $(Build.SourcesDirectory)/src/csharp/Microsoft.Spark/bin/$(buildConfiguration)
+        sourceFolder: $(Build.SourcesDirectory)/artifacts/packages/$(buildConfiguration)/Shipping
         contents: |
-          **/*.dll
-          **/*.pdb
-        targetFolder: $(Build.ArtifactStagingDirectory)/BuildArtifacts/src/csharp/Microsoft.Spark/bin/$(buildConfiguration)
+          **/*.nupkg
+          **/*.snupkg
+        targetFolder: $(Build.ArtifactStagingDirectory)/BuildArtifacts/artifacts/packages/$(buildConfiguration)/Shipping
```

Comment:
If the target folder is `$(Build.ArtifactStagingDirectory)/BuildArtifacts` and we make the `contents` be: `**/Shipping/**/*.nupkg` and `**/Shipping/**/*.snupkg` and then the sourceFolder being `(Build.SourcesDirectory)` it should preserve the folder structure, right?

Reply:
Yes I think so. I can try changing it that way tomorrow morning.

Reply:
Not required to merge but it’s a nice cleanup.